src/App.tsx

```diff
@@ -5,7 +5,18 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (newMovie: Movie) => {
+    const isNewMovieInList
+      = movies.some(item => item.imdbId === newMovie.imdbId);
```

Comment:
Looks better in my opinion
```suggestion
    const isNewMovieInList = movies.some(
      item => item.imdbId === newMovie.imdbId
    );
```